google/cloud/storage/examples/storage_object_acl_samples.cc

```diff
@@ -87,6 +87,24 @@ void CreateObjectAcl(gcs::Client client, int& argc, char* argv[]) {
   (std::move(client), bucket_name, object_name, entity, role);
 }
 
+void DeleteObjectAcl(gcs::Client client, int& argc, char* argv[]) {
+  if (argc < 3) {
+    throw Usage{"delete-object-acl <bucket-name> <object-name>"};
+  }
+  auto bucket_name = ConsumeArg(argc, argv);
+  auto object_name = ConsumeArg(argc, argv);
+  auto entity = ConsumeArg(argc, argv);
+  //! [delete object acl] [START storage_delete_file_acl]
+  [](google::cloud::storage::Client client, std::string bucket_name,
+     std::string object_name, std::string entity) {
```

Comment:
I would first question what an entity is here.

Reply:
Please be more blunt.  What did I do wrong?  Is it that the meaning of `entity` is unclear?  If so, I agree, but that is what the API calls it and I have no better ideas for the name. `username` is wrong, because it can be a team, or group, or other... well entity I guess.

Reply:
Reviewing other samples this is okay: [storage/docs/access-control/create-manage-lists#set-an-acl](https://cloud.google.com/storage/docs/access-control/create-manage-lists#set-an-acl).

Ignore comment as there isn't a clear alternative besides copying documentation into the code sample.